src/blackboxTests/src/test/kotlin/org/vaccineimpact/api/blackboxTests/DemographicTests.kt

```diff
@@ -0,0 +1,70 @@
+package org.vaccineimpact.api.blackboxTests
+
+import com.beust.klaxon.json
+import org.assertj.core.api.Assertions
+import org.junit.Test
+import org.vaccineimpact.api.blackboxTests.helpers.validate
+import org.vaccineimpact.api.db.JooqContext
+import org.vaccineimpact.api.db.direct.*
+import org.vaccineimpact.api.models.permissions.PermissionSet
+import org.vaccineimpact.api.test_helpers.DatabaseTest
+
+class DemographicTests: DatabaseTest()
+{
+    val requiredPermissions = PermissionSet("*/can-login", "*/touchstones.read", "*/demographics.read")
+    val touchstoneId = "touchstone-1"
+    val touchstoneName = "touchstone"
+    val touchstoneVersion = 1
+
+    @Test
+    fun `can get demographic stat types for touchstone`()
+    {
+        validate("/touchstones/$touchstoneId/demographics/") against "Demographics" given {
+            setUpSupportingTables(it)
+            setUpTouchstone(it)
+            addPopulation(it)
+
+        } requiringPermissions {
+            requiredPermissions
+        } andCheckArray {
+            Assertions.assertThat(it.count()).isEqualTo(1)
+            Assertions.assertThat(it).contains(json { obj(
+                    "id" to "tot-pop",
+                    "name" to "tot-pop descriptive name",
+                    "source" to "UNWPP2015 descriptive name",
+                    "countries" to array(_countries.sortedBy { it }),
+                    "gender_is_applicable" to false
+            )})
+        }
+    }
+
+    private var _countries: List<String> = listOf()
+    private var _sourceIds: List<Int> = listOf()
+    private var _sources: List<String> = listOf("UNWPP2015", "UNWPP2017")
+    private var _variantIds: List<Int> = listOf()
+    private var _variants = listOf("low", "medium", "high")
+    private var _units: List<Int> = listOf()
```

Comment:
Also, a more Kotlin-y way might be this. I mean, what you have is totally fine, I was just wondering what the 'var-less' approach would be...

```
private data class SupportingTables(db: JooqContext)
{
    val countries = db.generateCountries(3)
    val sources = listOf("UNWPP2015", "UNWPP2017")
    val sourceIds = db.generateDemographicSources(sources)
    val variants = listOf("low", "medium", "high")
    val variantIds = it.generateDemographicVariants(variants)
    val units = it.generateDemographicUnits()

    init
    {
        it.generateGenders()
    }
}
```

And then:
```
validate("/touchstones/$touchstoneId/demographics/") against "Demographics" given {
            val supportingTables = SupportingTables(it)
            setUpTouchstone(it, supportingTables)
            addPopulation(it, supportingTables)
}
...
```